spec/requests/query_controller_spec.rb

```diff
@@ -304,7 +318,7 @@ def run_query(id, params = {})
             DataExplorer.send(:remove_const, "QUERY_RESULT_MAX_LIMIT")
             DataExplorer.const_set("QUERY_RESULT_MAX_LIMIT", 2)
 
-            ids = Post.order(:id).pluck(:id)
+            _ids = Post.order(:id).pluck(:id)
 
             query = make_query <<~SQL
```

Comment:
```suggestion
            query = make_query <<~SQL
```

seems like this line isn't needed? or at least we don't need to assign it to a variable?